indico/modules/events/editing/client/js/components/FileManager/reducer.js

```diff
@@ -0,0 +1,110 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import _ from 'lodash';
+import {combineReducers} from 'redux';
+import * as actions from './actions';
+
+export default combineReducers({
+  fileTypes: (state = [], action) => {
+    const fileType = _.find(state, {id: action.fileTypeId});
```

Comment:
We don't have to always use lodash if ES supports sth like that out of the box :P

```javascript
state.find(type => type.id === action.fileTypeId);
```

Reply:
See my comment below. I don't agree that it's just 'out of the box'. `lodash` is more convenient IMHO.

Reply:
It is a matter of preference. I e.g. prefer to use something that is provided directly by the engine.